test/test_time_series.jl

```diff
@@ -82,21 +78,15 @@ end
     other_time = initial_time + resolution
     name = "test"
     horizon = 24
-    data_vec = Dict(initial_time => ones(horizon), other_time => ones(horizon))
-    data_tuple =
-        Dict(initial_time => tuple(ones(horizon)...), other_time => tuple(ones(horizon)...))
```

Comment:
I don't think we need to support this.  @sourabhdalvi @jd-lara Please comment if you disagree.

Reply:
Not anymore, I added it to test the case where we convert to `Vector{Float64}`, given these changes this needs to be eliminated.